src/seedu/addressbook/commands/HelpCommand.java

```diff
@@ -17,6 +17,7 @@ public HelpCommand() {}
     public CommandResult execute() {
         return new CommandResult(
                 AddCommand.MESSAGE_USAGE
+                + "\n" + AddTagCommand.MESSAGE_USAGE
```

Comment:
@okkhoy 
Somehow adding this line cause the checks to fail: "The Travis CI build failed".
Why is that so?